examples/address_generator.py

```diff
@@ -15,7 +15,6 @@
 
 from iota import Iota, __version__
 from iota.crypto.addresses import AddressGenerator
-from iota.crypto.types import Seed
```

Comment:
❕ This import is used on line 60, as part of a type hint:

https://github.com/iotaledger/iota.lib.py/blob/develop/examples/address_generator.py#L59-L60

Reply:
Ah good catch, should have looked closed. Will drop this patch.